client/app/components/Modal/Modal.scss

```diff
@@ -33,8 +33,10 @@
     height: 60vh;
     z-index: $z-index-front;
     @media screen and (max-width: $small) {
+      position: absolute;
       width: 100vw;
-      height: 100vh;
+      height: 80vh;
```

Comment:
So we still want our modal to take up the full-screen height. Could you try 100% instead and see how that works?